authentication/azure/auth.go

```diff
@@ -41,6 +41,8 @@ func NewEnvironmentSettings(resourceName string, values map[string]string) (Envi
 	case "cosmosdb":
 		// Azure Cosmos DB (data plane)
 		es.Resource = "https://" + azureEnv.CosmosDBDNSSuffix
+	case "serviceBus":
```

Comment:
Nit: this should probably be all lowercased for consistency with the other identifiers

Reply:
done